packages/client/hmi-client/src/components/models/tera-model.vue

```diff
@@ -596,6 +597,8 @@ watch(
 			model.value = await getModel(props.assetId);
 			if (model.value && model.value.name === 'Bucky') {
 				amr.value = bucky;
+				console.log('Test case hit');
```

Comment:
Log statements

Reply:
LOL Thanks theres definitely a lot of cleanup to be done on my end